src/main/webapp/WEB-INF/tags/instructor/feedbackSessionsNewForm.tag

```diff
@@ -0,0 +1,519 @@
+                            </label>
+                            <input type="checkbox" checked="checked"
+                                name="<%= Const.ParamsNames.FEEDBACK_SESSION_SENDREMINDEREMAIL %>"
+                                id="<%= Const.ParamsNames.FEEDBACK_SESSION_SENDREMINDEREMAIL %>_closing"
+                                value="<%= EmailType.FEEDBACK_CLOSING.toString() %>">
+                        </div>
+                    </div>
+                    <div class="col-sm-4"
+                        title="<%= Const.Tooltips.FEEDBACK_SESSION_SENDPUBLISHEDEMAIL %>"
+                        data-toggle="tooltip"
+                        data-placement="top">
+                        <div class="checkbox">
+                            <label for="<%= Const.ParamsNames.FEEDBACK_SESSION_SENDREMINDEREMAIL %>_published">
+                                Results published announcement
+                            </label>
+                            <input type="checkbox" checked="checked"
+                                name="<%= Const.ParamsNames.FEEDBACK_SESSION_SENDREMINDEREMAIL %>"
+                                id="<%= Const.ParamsNames.FEEDBACK_SESSION_SENDREMINDEREMAIL %>_published"
+                                value="<%= EmailType.FEEDBACK_PUBLISHED.toString() %>">
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="form-group">
+            <div class="row">
+                <div class="col-md-offset-5 col-md-3">
+                    <button id="button_submit" type="submit" class="btn btn-primary"
+                        ${newForm.submitButtonDisabledAttribute}>
+                        Create Feedback Session
+                    </button>
+                </div>
+            </div>
+        </div>
+        <c:if test="${empty data.newForm.courses}"> 
+            <div class="row">
+                <div class="col-md-12 text-center">
+                    <b>You need to have an active(unarchived) course to create a session!</b>
+                </div>
+            </div>
+        </c:if>
+        <input type="hidden"
+            name="<%= Const.ParamsNames.USER_ID %>"
+            value="${data.account.googleId}">
+    </form>
+    <form style="display:none;" id="ajaxForSessions" class="ajaxForSessionsForm"
+        action="<%= Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE %>">
+        <input type="hidden"
+            name="<%= Const.ParamsNames.USER_ID %>"
+            value="${data.account.googleId}">
+        <input type="hidden"
+            name="<%= Const.ParamsNames.IS_USING_AJAX %>"
+            value="on">
+        <c:if test="${ data.newForm.feedbackSessionNameForSessionList != null && data.newForm.courseIdForNewSession != null }">
+            <input type="hidden"
+                name="<%= Const.ParamsNames.FEEDBACK_SESSION_NAME %>"
+                value="${data.newForm.feedbackSessionNameForSessionList }">
```

Comment:
If you passed `data.newForm` as `newForm` in here, there's no need to refer to `data.newForm` again.